data/json/mapgen/bridges.json

```diff
@@ -0,0 +1,164 @@
+      "=": "t_guardrail_bg_dp",
+      "u": "t_ramp_up_low",
+      "U": "t_ramp_up_high",
+      "d": "t_ramp_down_high",
+      "D": "t_ramp_down_low",
+      "W": "t_sidewalk_ramp_down_high",
+      ">": "t_sidewalk_ramp_down_low",
+      "<": "t_sidewalk_ramp_up_high",
+      "w": "t_sidewalk_ramp_up_low",
+      "#": "t_concrete_wall"
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "bridge" ],
+    "object": {
+      "fill_ter": "t_water_moving_dp",
+      "rows": [
+        "~~~~~~~~s######s~~~~~~~~",
+        "~~~~~~~~ss####ss~~~~~~~~",
+        "~~~~~~~~~ss##ss~~~~~~~~~",
+        "~~~~~~~~~~ssss~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~~~~~~~~~~~~~~~",
+        "~~~~~~~~~~ssss~~~~~~~~~~",
+        "~~~~~~~~~ss##ss~~~~~~~~~",
+        "~~~~~~~~ss####ss~~~~~~~~",
+        "~~~~~~~~s######s~~~~~~~~"
+      ],
+      "palettes": [ "bridge_palette" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "bridge_road" ],
+    "object": {
+      "fill_ter": "t_open_air",
+      "rows": [
+        "  =_................_=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_................_=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_................_=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_................_=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_................_=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_................_=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  ",
+        "  =_.......::......._=  "
+      ],
+      "palettes": [ "bridge_palette" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "bridgehead_ground" ],
+    "object": {
+      "fill_ter": "t_water_moving_dp",
+      "rows": [
+        "ss=_................_=ss",
+        "ss=_.......::......._=ss",
+        "ss=_.......::......._=ss",
+        "ss=_.......::......._=ss",
+        "ss=_................_=ss",
+        "ss=wuuuuuuuuuuuuuuuuw=ss",
+        "ss<UUUUUUUUUUUUUUUUU<=ss",
```

Comment:
Something looks off here:

![image](https://user-images.githubusercontent.com/16213433/84904448-fb4f3f80-b0b7-11ea-8f23-39229befebed.png)

Reply:
grrr, argh.

Thanks for catching that.